### PR TITLE
Dice section

### PR DESCRIPTION
**Pull request by agude**

### CODE REVIEW

**Comment by agude** [comment]
Things I'd like to do:

- [ ] Add in a diagram of a aspect with evocation
- [ ] Write some examples

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
 
 ## Rolling the Dice
 
+Dice are an important part of {GameName} because they add unpredictability to the story, but that doesn't mean you need to roll for every little thing.
```
Comment (charles-uno) [inline _chapters/00-outline.md:211]: > but that doesn't mean you need to roll for every little thing

Not sure about "unpredictability" but I love this

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
...
+<!-- Active is just harming people? -->
+Active opposition is determined by another player's roll.
+The difference between your result and the opposition is called the shift.
+For example, if your roll equals the opposition, you have 0 shifts.
+If you beat the opposition by 1, you have 1 shift.
+Shifts provide bonuses or penalties depending on the type of roll.
+
+Players can roll to:
+
+- Create an scene aspect with a free invocation
```
Comment (charles-uno) [inline _chapters/00-outline.md:240]: an -> a

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
...
+- Destroy an scene aspect
```
Comment (charles-uno) [inline _chapters/00-outline.md:241]: - Create, modify, or destroy an aspect

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
...
+- Inflict stress on another character (typically an NPC)
+- Defend against someone inflicting stress on them
```
Comment (charles-uno) [inline _chapters/00-outline.md:243]: Option 1:

I am rolling to inflict stress

vs

I am rolling to punch this guy. Success. Now I decide how I want to reflect it mechanically. Stress or aspect

File: _chapters/00-outline.md [hunk 5]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
...
+| Die Face  | Value |
+|:---------:|:-----:|
+| {{neg}}   | -1    |
+| {{zero}}  | 0     |
+| {{pos}}   | +1    |
+
+Compare the result to the opposition.
+Opposition is either passive or active.
+<!-- Passive is just creating an aspect -->
+Passive opposition is a target number set by the GM.
```
Comment (charles-uno) [inline _chapters/00-outline.md:230]: I don't love that rolls are sometimes opposed and sometimes not. Can we standardize? I personally like having only players roll, but we could also have players and GM both always roll

File: _chapters/00-outline.md [hunk 6]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
...
+- Defend against someone inflicting stress on them
+
+### Creating An Aspect
+
+Sometimes the aspects describing a scene aren't the ones you want.
+Your character can create new aspects or destroy existing ones with a role.
+To do so, the player states their intention and the GM sets a difficulty from 0 (easy) to 6 (hard).
+The player rolls their dice, sums the result, and adds any appropriate character aspect value (0 if none of their aspects fit the situation).
+If this total meets or beats the GM's difficulty, they succeed.
+When creating an aspect, the player gets one free invocation; they get an additional invocation if they succeed with style by beating the target number by 2 or more.
```
Comment (charles-uno) [inline _chapters/00-outline.md:252]: Does this keep scaling? If you beat it by 4 do you get another free invocation?

Comment (charles-uno) [inline _chapters/00-outline.md:252]: Does the free invocation belong to the player or the aspect? Can another PC use it? Can the GM use it against you?

Also maybe add a TODO: show what this looks like. Aspect written on the whiteboard with some checkboxes

File: _chapters/00-outline.md [hunk 7]
```diff
@@ -208,6 +208,51 @@ This dichotomy gives each PC quantitative strengths and weaknesses, while still
 
 ## Rolling the Dice
 
+Dice are an important part of {GameName} because they add unpredictability to the story, but that doesn't mean you need to roll for every little thing.
+As a general rule, the GM should prompt players for a roll in situations where a competent hero might reasonably fail _and_ failure adds an interesting twist to the story.
+For example, a `Swashbuckling Genius` probably doesn't need to roll to fend off an attack by a `Nameless Deck-Hand`...
+but she might roll to subdue him before his friends escape, or to recognize him as a `Student of the Blade Dancers`.
+
+Dice give aspects mechanical power.
+When you are called upon to roll, figure out with your group which of your character's aspects is most appropriate and note the rating.
```
Comment (charles-uno) [inline _chapters/00-outline.md:217]: Have been percolating on this. Can concept and trouble both be +2? Just one less thing to worry about. Basically you always have a free invoke on one of your aspects

**Review by charles-uno — APPROVED** [review]

**Comment by charles-uno** [comment]
I see it's marked as WIP, but seems fine to merge this and keep iterating in a new commit

File: _chapters/00-outline.md (hunk 6 above)
Comment (agude) [reply _chapters/00-outline.md:252]: Let's start with just 2 or more and see.

File: _chapters/00-outline.md (hunk 7 above)
Comment (agude) [reply _chapters/00-outline.md:217]: We could! Let's discuss tomorrow.

**Pull request merged**